snapcraft.yaml

```diff
@@ -294,14 +294,14 @@ parts:
   pango:
     after: [ harfbuzz, meson-deps ]
     source: https://gitlab.gnome.org/GNOME/pango.git
-    source-tag: '1.52.2'
+    source-tag: '1.54.0'
     source-depth: 1
     plugin: meson
     meson-parameters:
       - --prefix=/usr
       - -Doptimization=3
       - -Ddebug=true
-      - -Dinstall-tests=false
+      - -Dbuild-testsuite=false
```

Comment:
The change was https://gitlab.gnome.org/GNOME/pango/-/commit/927abcc which also added -Dbuild-examples, could you set that one to false?

Reply:
addressed!